README.md

```diff
@@ -1,34 +1,85 @@
-# echo-client
 
-Client repo for the Echo App. It displays the notepad sync between all echoers to make the Echo App echo!
+# 📝 **Echo Client** 🌟
 
-## Client Events
+Welcome to the **Echo Client** repository! This innovative application transforms note-taking by enabling **real-time synchronization** among all users. Whether you’re collaborating on projects or sharing thoughts with friends, **Echo** ensures that your notes resonate across the digital landscape, making every conversation impactful! 🌐✨
 
-- Emits
-  1. init -> a connection, with name and roomId (from url, or randomly generated)
-  2. \*echo -> sent whenever text is typed
-  3. leave -> this client left the room, that's it
-- Recieves
-  1. \*reverb -> recieves data and updates the notepad
-  2. joined -> new client joined the room
-  3. left -> someone left the room, recieves name and current members
+---
 
-NOTE: \* marked events are the main heroes
+## 📖 **Table of Contents**
```

Comment:
ToC is not something that we want for small READMEs. GitHub generates a ToC by default